Doc/Makefile

```diff
@@ -12,7 +12,11 @@ PAPER        =
 SOURCES      =
 DISTVERSION  = $(shell $(PYTHON) tools/extensions/patchlevel.py)
 
-ALLSPHINXOPTS = -b $(BUILDER) -d build/doctrees -D latex_elements.papersize=$(PAPER) \
```

Comment:
@zware because PAPER maybe unmodified as is the case when one does directly `make latex` with this Makefile, contrarily to the `make dist` case where PAPER is set explicitely for the a4 latex and letter latex builds.